src/software/backend/robot_status.cpp

```diff
@@ -0,0 +1,35 @@
+#include "software/backend/robot_status.h"
+
+TbotsRobotMsg convertRobotStatusToTbotsRobotMsg(RobotStatus* robot_status)
+{
+    // Insufficient ChipperKickerStatus, DriveUnits, and NetworkStatus info
+    TbotsRobotMsg robot_msg;
+
+    robot_msg.set_robot_id(robot_status->robot);
+
+    auto break_beam_msg = std::make_unique<BreakBeamStatus>();
+    break_beam_msg->set_ball_in_beam(robot_status->ball_in_beam);
+    break_beam_msg->set_break_beam_reading((float)robot_status->break_beam_reading);
```

Comment:
according to the code style guide, change this cast to static_cast

Reply:
Done.